src/coreneuron/io/reports/report_handler.cpp

```diff
@@ -380,6 +380,18 @@ VarsToReport ReportHandler::get_lfp_vars_to_report(const NrnThread& nt,
                 }
             }
         }
+        auto mech_id_seclamp = nrn_get_mechtype("SEClamp");
+        Memb_list* ml_seclamp = nt._ml_list[mech_id_seclamp];
+        if (ml_seclamp) {
+            for (int j = 0; j < ml_seclamp->nodecount; j++) {
+                auto segment_id = ml_seclamp->nodeindices[j];
+                if ((nodes_to_gids[segment_id] == gid)) {
+                    double* var_value =
+                        get_var_location_from_var_name(mech_id_seclamp, "i", ml_seclamp, j);
+                    summation_report.currents_[segment_id].push_back(std::make_pair(var_value, -1));
+                }
+            }
+        }
```

Comment:
`nitpick`: `IClamp` and `SEClamp` has same code. Maybe you could avoid duplication? (e.g. most of the code can go into a lambda?)